metadata-ingestion/src/datahub/sql_parsing/sql_parsing_aggregator.py

```diff
@@ -848,7 +847,6 @@ def add_preparsed_query(
             query_fingerprint = get_query_fingerprint(
                 parsed.query_text,
                 platform=self.platform.platform_name,
-                fast=True,
```

Comment:
we'll need to revisit this - we want fast fingerprinting for snowflake, and slow parsing-based for all other platforms